package.json

```diff
@@ -48,7 +48,7 @@
   "scripts": {
     "docs": "repo-tools exec -- jsdoc -c .jsdoc.js",
     "generate-scaffolding": "repo-tools generate all --except lib_readme",
-    "lint": "cd samples/ && npm link ../ && repo-tools lint --cmd eslint -- src/ samples/ system-test/ test/",
+    "lint": "cd samples/ && npm install && npm link ../ && cd .. && repo-tools lint --cmd eslint -- src/ samples/ system-test/ test/",
```

Comment:
I think you should just change this to:

```
"lint": "repo-tools lint --cmd eslint -- src/ samples/ system-test/ test/",
```

which is what all the other repos use.